code/controllers/subsystem.dm

```diff
@@ -169,23 +169,25 @@
 	return time
 
 //hook for printing stats to the "MC" statuspanel for admins to see performance and related stats etc.
-/datum/controller/subsystem/stat_entry(msg)
+/datum/controller/subsystem/proc/stat_entry_legacy()
```

Comment:
why does this exist ?

Reply:
primarily just for legacy purposes, it *can* be removed safely (though I just woke up so let me double check)

Reply:
this just in: it serves no purpose than to represent my lack of remembering to delete things